packages/vsce/src/commands/inspectResourceCommandUtils.ts

```diff
@@ -163,8 +160,11 @@ async function loadResources(
   const resources = await resourceContainer.fetchNextPage();
 
   if (resources.length === 0) {
-    const hrn = resourceTypes.map((type) => type.humanReadableNameSingular).join(" or ");
-    const message = CICSMessages.CICSResourceNotFound.message.replace("%resource-type%", hrn).replace("%resource-name%", resourceName).replace("%region-name%", resourceContext.regionName);
+    const hrn = resourceTypes.map((type) => l10n.t(type.humanReadableNameSingular)).join(l10n.t("Or"));
```

Comment:
You've lost the space and capitalised the `Or` here, resulting in this error message

<img width="242" height="40" alt="Image" src="https://github.com/user-attachments/assets/12e7e38e-9845-4501-a28e-7b713d3635df" />